console-subscriber/examples/uds.rs

```diff
@@ -0,0 +1,23 @@
+use std::time::Duration;
```

Comment:
what does this example do if it's run on a non-Unix OS? if the `ConsoleLayer` builder emits a nice error here, that's sufficient, but it's definitely important that something in the example makes it clear that this won't work on Windows etc.

in any case, it might be nice to add a comment here like
```suggestion
//! Demonstrates serving the console API over a [Unix Domain Socket] (UDS)
//! connection, rather than over TCP.
//!
//! Note that this example only works on Unix operating systems that 
//! support UDS, such as Linux, BSDs, and macOS.
//!
//! [Unix Domain Socket]: https://en.wikipedia.org/wiki/Unix_domain_socket
use std::time::Duration;
```

Reply:
Added the comment! I don't have a Windows machine so I can't easily test what happens if you try this on Windows.